addons/pos_sms/models/pos_order.py

```diff
@@ -0,0 +1,20 @@
+from odoo import models
+
+
+class PosOrder(models.Model):
+    _inherit = 'pos.order'
+
+    def action_sent_message_on_sms(self, name, partner, ticket_image):
```

Comment:
why pass `name` and `ticket_image` if they are not  used ?

Reply:
Hi @vlst-odoo ,
Thanks for your suggestion,

Instead of using `async sendToCustomer(orderPartner, methodName)` from point_of_sale, we can directly call a Python method using ORM like : `this.pos.data.call("pos.order", methodName, [])`.